rust_src/src/windows.rs

```diff
@@ -672,6 +672,19 @@ pub fn window_live_p(object: Option<LispWindowRef>) -> bool {
     object.map_or(false, |m| m.is_live())
 }
 
+/// Return new pixel size of window WINDOW.
+/// WINDOW must be a valid window and defaults to the selected one.
+/// The new pixel size of WINDOW is the value set by the last call of
+/// `set-window-new-pixel' for WINDOW.  If it is valid, it will be shortly
+/// installed as WINDOW's pixel height (see `window-pixel-height') or pixel
+/// width (see `window-pixel-width').
+
+#[lisp_fn]
```

Comment:
This is missing `min = "0"`. 

`DEFUN ("window-new-pixel", Fwindow_new_pixel, Swindow_new_pixel, 0, 1, 0,`

The 0, 1, 0 translate to: "min 0, max 1, not interactive".

In rust_src/admin there is a python script called rustize-emacs which helps automate these ports.
```suggestion
#[lisp_fn(min = "0")]
```

Reply:
@AshtonKem the min is usually set to the number of arguments the function takes. In this case it would have been min=1.